src/mechanics.asm

```diff
@@ -243,8 +267,8 @@ TRB $0F6A
 RTS
 
 Swimming:
-LDA #$23
-TRB $0DB9
+;LDA #$23	...Wait, Luigi Can't Look Up in the Water!?
+;TRB $0DB9	...But, Why?!
```

Comment:
This i the one change I'm not sure about - I believe this code is only called while luigi is swimming properly, i.e. not touching the ground. In that case it would make sense to reset the lookup up, ducking, etc bits.